web-common/src/main/resources/META-INF/resources/view/login.jsp

```diff
@@ -19,6 +19,9 @@
 <body>
 <%@ include file="header.jsp" %>
 <div id="loginContainer" class="container">
+    <div class="confidential-message text-center" > 
+        <img src="images/security_warning_pt.${projectBuildNumber}.png" class="img-fluid" alt="Responsive image"/>
```

Comment:
This is an example of why images are not good, you are including the Portuguese image meaning the english version won't work

Reply:
Done